.github/workflows/testadapters.yaml

```diff
@@ -105,26 +118,23 @@ jobs:
               if [ "$svcstatus" = "<pending>" ];then
                  exit 1
               fi
-
       # Reset meshery's address to the external IP in config.yaml such that mesheryctl can reach it
-      - name: Reset meshery address
+      - name: Reset Meshery address
         run: |
           kubectl get svc -n meshery 
           svcip="http://$(kubectl get svc -n meshery | grep "meshery " | tr -s ' ' | cut -d " " -f 3):9081"
           echo "this is $svcip"
           config='{"contexts":{"local":{"endpoint":'$svcip',"token":"Default","platform":"kubernetes","adapters":[],"channel":"stable","version":"latest"}},"current-context":"local","tokens":[{"location":"auth.json","name":"Default"}]}'
           echo $config  > ~/.meshery/config.yaml
-
-
       - name: Download patternfile
         uses: actions/download-artifact@v2
         with:
           name: patternfile
       # This pattern will try to install  service mesh
       - name: Pattern apply (Install service mesh)
         run: |
+            cat ./${{ inputs.patternfile_name }}
             mesheryctl pattern apply -f ./${{ inputs.patternfile_name }}
-
       # Wait for sometime to get some logs
       - name: Sleep for 30 sec
         run: sleep 30s
```

Comment:
Ugh

Reply:
Least I can do is reduce them to 10 seconds. I want to make sure, I get all the logs properly and it takes time for all the logs to come up.

Reply:
The problem with sleeping isn't time itself, but know how much time to sleep. Yes, it's inefficient and not ideal. The real issue is that this approach inevitably leads to bugs.  

find something else to poll; something else to check to determine when to move forward instead of sleeping.